### PR TITLE
Minor improvement for doc.

### PR DESCRIPTION
**Pull request by csukuangfj**

### CODE REVIEW

File: src/openMVG/numeric/poly.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace openMVG
 /**
 * @brief Solve roots of a cubic polynomial
 * \f$ x^3 + a x^2 + b x + c = 0 \f$
-* @param a Coefficient of cubic parameter
+* @param a Coefficient of quadratic parameter
```
Comment (pmoulon) [inline src/openMVG/numeric/poly.h]: This a a cubic polynomial

**Comment by pmoulon** [comment]
Thx. Appreciated.
I made only one comment. Appreciated if you can fix it.

File: src/openMVG/numeric/poly.h (hunk 1 above)
Comment (csukuangfj) [reply src/openMVG/numeric/poly.h]: @pmoulon 
thank you for the comment.

a is the coefficient of x^2, which is a quadratic term. Shouldn't a be the quadratic
parameter?

Comment (pmoulon) [reply src/openMVG/numeric/poly.h]: You're right. I'm sorry. I read to fast ;-)

**Comment by pmoulon** [comment]
Thx merged by https://github.com/openMVG/openMVG/commit/01cb080b5af6de164d94c19caf90ef93a82c10dd

**Pull request closed without merging**